apache-flink-+-tidb-a-scale-out-real-time-data-warehouse-for-analytics-within-seconds.md

```diff
@@ -103,6 +103,11 @@ If you want to store MySQL change logs or other data sources in Kafka for Flink
 
 In [TiDB 4.0.8](https://docs.pingcap.com/tidb/stable/release-4.0.7), you can connect TiDB to Flink through the [TiCDC Open Protocol](https://docs.pingcap.com/zh/tidb/stable/ticdc-open-protocol). In later versions, TiCDC will support the canal-json output format for Flink's use.
 
+<div class="trackable-btns">
+  <a href="/download" onclick="trackViews('Apache Flink + TiDB: A Scale-Out Real-Time Data Warehouse for Analytics Within Seconds', 'download-tidb-btn-middle')"><button>Download TiDB</button></a>
+  <a href="https://share.hsforms.com/1e2W03wLJQQKPd1d9rCbj_Q2npzm" onclick="trackViews('Apache Flink + TiDB: A Scale-Out Real-Time Data Warehouse for Analytics Within Seconds', 'subscribe-blog-btn-middle')"><button>Subscribe to Blog</button></a>
```

Comment:
Please use `contact us` instead of `subscribe blog`

Reply:
ok